roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,38 @@
+- publisher:
+    name: bonnyci-logs
+    publishers:
+      - scp:
+          site: bonnyci-scp
+          files:
+            - target: 'logs/$LOG_PATH'
+              source: 'logs/**'
+              keep-hierarchy: true
+              copy-after-failure: true
+
+- job-template:
+    name: 'dib-build-{pipeline}'
+    id: 'dib-build-pipeline-{pipeline}'
+    node: '{node}'
+    builders:
+      shell: |
+        virtualenv $HOME/dib-virtualenv
+        . $HOME/dib-virtualenv/bin/activate
+        pip install diskimage-builder
+
+        git clone https://github.com/BonnyCI/hoist.git $HOME
+        export ELEMENTS_PATH=$HOME/hoist/roles/nodepool/files/etc/nodepool/elements/nodepool
+        disk-image-create -a amd64 -o test-ubuntu vm ubuntu
```

Comment:
So we want to actually test an image build with all the elements we have in-repo.  The build command we actually end up running in production is:

disk-image-create -x -t qcow2 --checksum --no-tmpfs --qemu-img-options 'compat=0.10' -o /opt/nodepool/images/ubuntu-xenial-0000000037 ubuntu-minimal vm simple-init growroot openssh-server devuser haveged pip-and-virtualenv nodepool bonnyci-nodepool

Not sure if want to hard-code that into the job definition here,  parse the elements-to-test out of a 'nodepool_diskimages' variable somewhere in inventory, or just try to build an image with all elements listed in $HOME/hoist/roles/nodepool/files/etc/nodepool/elements/nodepool ?